index.html

```diff
@@ -50,6 +50,147 @@ <h3 class="helpingtext">LET'S CONNECT</h3>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>
+    <article class="Box">
+        <img class="box-image" src="req/multi_2.png" alt="multi_2">
+        <div class="content">
+            <h2 class="content-title">Multi-Post Stories</h2>
+            <div class="text-title">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="client-skills">BACK END DEV</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="period">2015</a>
+            </div>
+            <p class="content-text">A daily selection Of privately personalized reads; no accounts or sign-ups required</p>
+            <ul class="tags">
+                <li class="style">html</li>
+                <li class="style">css</li>
+                <li class="style">javascript</li>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>  
+</section>
+
+
+<section class="About-container">
+    <div class="top">
+        <h2 class="about-head">About </br> Myself</h2>
+        <p class="text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3 class="helpingtext">LET'S CONNECT</h3>
+        <div class="belowtext">
+            <ul >
+                <li>
+                    <a href="#"><img src="req/twitter.png" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/Linkedin.png" alt="linkedin"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/M.png" alt="M"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/Github.png" alt="Github"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/V.png" alt="V"></a>
+                </li>
+            </ul>
+        </div>
+        <a href="#" class="link">Get my resume</a>
+    </div>
+    <ul class="about-skills">
+        <li class="lang-detail">
+        <h3 class="lang-head">Languages</h3>
+        <i class="arrow down"></i>
+        </li>
+        <ul class="lang-names">
+            <li class="lang-container">
+                <img class="lang-img" src="req/js.png" alt="javascript">
+                <p class="lang">javascript</p>
+            </li>
+            <li class="lang-container">
+                <img class="lang-img" src="req/html.png" alt="html">
+                <p class="lang">HTML</p>
+            </li>
+            <li class="lang-container">
+                <img class="lang-img" src="req/css.png" alt="css">
+                <p class="lang">CSS</p>
+            </li>
+        </ul>
```

Comment:
I did very hard work .kindly approved it because I have many tasks left and it's my 4th review request. if anything else needed to change I will fix in next project.